modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js

```diff
@@ -21,9 +22,11 @@ import {
 	getAccountSubscriptionsTerms,
 } from '../../../../../common/services/liferay/graphql/queries';
 import getActivationStatusDateRange from '../../../../../common/utils/getActivationStatusDateRange';
+import {ALERT_UPDATE_DXP_CLOUD_STATUS} from '../../../containers/DXPActivationKeysTable/utils/constants/alertUpdateDXPCloudStatus';
```

Comment:
This utils should be exported from utils/constants and not being imported directly. Import them at the root of constants and import this root here
```suggestion
import {ALERT_UPDATE_DXP_CLOUD_STATUS} from '../../../containers/DXPActivationKeysTable/utils/constants';
```

Reply:
Ok, man. Thks!

Reply:
OK!